src/components/controls/link/km-add-file.vue

```diff
@@ -0,0 +1,69 @@
+<template>    
+    <div class="scbd-common km-add-file" >
+        <select-file-button  :multiple="props.multiple" @on-file-selected="receiveFile"> <slot name="file-button-label">+ Add Files</slot></select-file-button> 
+              
+        <file-upload-editor ref="fileEditorRef" @on-close="onFileUploadEditorClose">
+            <template v-slot:modalTitle>
+            <slot name="link-dialog-title"   >        
+                Uploading file    
+            </slot>
+            </template>       
+        </file-upload-editor>
+        <slot name="links-view">
+            <km-view-links v-model="links"   @on-delete = "removeLink"   @on-edit = "editLink"></km-view-links>  
+        </slot>
+    </div>
+</template>
+<script setup>
+    import _ from "lodash";
```

Comment:
Import only functions which are used for eg
Import { map} from lodash

Reply:
fixed